package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@powsybl/diagram-viewer",
+  "version": "0.0.1",
+  "description": "Typescript library to integrate a powsybl svg diagram in a javascript project.",
+  "author": "powsybl team",
+  "homepage": "https://github.com/powsybl",
+  "license": "MPL",
+  "repository": "git://github.com/powsybl/powsybl-diagram-viewer.git",
+  "types": "dist/types.d.ts",
+  "source": "src/index.ts",
+  "main": "dist/main.js",
+  "module": "dist/module.js",
+  "files": [
+    "/dist"
+  ],
+  "scripts": {
+    "watch": "parcel watch",
+    "build": "parcel build",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
to implement ?

Reply:
Yes it's in progress

Reply:
A simple test has been implemented. getDimensionsFromSvg() isn't tested as Jest doesn't support svg manipulation in the DOM.